end-to-end-test/local/specs/core/patientview.screenshot.spec.js

```diff
@@ -6,16 +6,46 @@ var assertScreenShotMatch = require('../../../shared/lib/testUtils')
     .assertScreenShotMatch;
 
 const CBIOPORTAL_URL = process.env.CBIOPORTAL_URL.replace(/\/$/, '');
-const patienViewUrl =
+const patientViewUrl =
     CBIOPORTAL_URL + '/patient?studyId=teststudy_genepanels&caseId=patientA';
+const ascnPatientViewUrl =
+    CBIOPORTAL_URL + '/patient?studyId=ascn_test_study&caseId=FAKE_P001';
 
 describe('patient view page', function() {
+    describe('mutation table for study with ascn data', () => {
+        beforeEach(() => {
+            goToUrlAndSetLocalStorage(ascnPatientViewUrl);
+            waitForPatientView();
+        });
+
+        it('ascn columns are displayed', () => {
```

Comment:
Convention for the `it` function is that it reads like a proper sentence, so while `it ascn columns are displayed` is not a proper sentence `it displays ascn columns` is. This comment applies most of the `it` functions introduced in this PR.

Reply:
@onursumer I'll update this today!